src/nimblepkg/packageinfo.nim

```diff
@@ -199,8 +200,15 @@ proc fetchList*(list: PackageList, options: Options) =
                 priority = LowPriority)
 
       try:
-        let client = newHttpClient(proxy = proxy)
+        let ctx = newSSLContext(options.disableSslCertCheck)
+        let client = newHttpClient(proxy = proxy, sslContext = ctx)
         client.downloadFile(url, tempPath)
+      except SslError:
+        let message = "Failed to verify the SSL certificate for " & url
+        display("Error:", message, Error, HighPriority)
+        # Terminate immediately: don't encourage ignoring the error
+        quit(QuitFailure)
```

Comment:
CC @genotrance, be aware that you should never quit in Nimble code but instead use this API.

```suggestion
        let message = "Failed to verify the SSL certificate for " & url
        raiseNimbleError(message, "Use --noSSLCheck to ignore this error.")
```

Reply:
Updated